### ISSUE TITLE
[Copy improvement] Update the message for accepting/declining a grant on Rafiki test wallet 

### ISSUE THREAD

**Issue by RabebOthmani** [interledger/web-monetization-extension#576]
### Context

This is a feedback from the beta testing ' I’ve never liked that it says “requesting to make a payment on your behalf” because there’s no payment being made (yet)' 

### Todos

- Update message on the accept/decline grant page

**Comment by raducristianpopa** [comment]
Not related to the extension - we do not have control on the wallet IdP (where users accept or decline grant).

**Comment by RabebOthmani** [comment]
I understand this is not work on the extension side but we will need to follow up with the rafiki team to make it happen thus the issue here for tracking. Also this is very specific to Rafiki. It would be interesting to see what messages other wallet providers have

**Comment by melissahenderson** [comment]
Yeah, just to add a little bit of context to this issue since the feedback was mine. I'm specifically referring to the grant consent screen in test network/test wallet. 

When testing the extension, I'm presented with the standard grant consent screen that I'd see if I were making a purchase from the boutique, for example. My suggestion is to have a separate grant consent screen that's shown when hooking the extension to a test wallet, so that the messaging to the end user is more clear. 

Today, it says "Melissa's payment pointer is requesting access to make a payment on your behalf....Total amount to debit: $5.000000000". It might be less confusing if the messaging was related to Web Monetization and what the user is doing --  linking their extension to their wallet to approve a budget.
